### PR TITLE
I107: Python bindings for GLONASS correlator

### PR DESCRIPTION
**Pull request by valeri-atamaniouk**
Part of I107 task. Required for GLONASS support in Peregrine.

### CODE REVIEW

**Comment by valeri-atamaniouk** [comment]
@adel-mamin @rge-exafore @gsmcmullin Please review

File: src/correlate.c [hunk 1]
```diff
@@ -144,6 +146,59 @@ void l2c_cm_track_correlate(const s8* samples, size_t samples_len,
                   I_E, Q_E, I_P, Q_P, I_L, Q_L, *num_samples);
 }
 
+/** Perform GLO CA correlation.
+ *
+ * \param samples          Samples array. One byte per sample.
+ * \param samples_len      Samples array size.
+ * \param code             L2C CM PRN code. One byte per chip: 10230 bytes long.
```
Comment (adel-mamin) [inline src/correlate.c]: L2C CM PRN -> GLO CA PRN
10230 -> 511

**Comment by adel-mamin** [comment]
Looks good to me.

**Comment by valeri-atamaniouk** [comment]
Replaced with private version.

**Pull request closed without merging**